.github/renovate.json

```diff
@@ -38,6 +38,10 @@
     {
       "matchPackageNames": ["react-from-dom"],
       "allowedVersions": "<=0.6.2"
+    },
+    {
```

Comment:
Our current Storybook version does not support anything about major version 2 for `node-fetch`.